src/dump.py

```diff
@@ -502,7 +502,7 @@ def scale_one(self,snap,x,y,z):
       h4inv = (h3*h5 - h1*h4) / (h0*h1*h2)
       h5inv = xy / (h0*h1)
       atoms = snap.atoms
-      if atoms != None:
+      if type(atoms) != types.NoneType:
```

Comment:
What does this enable or solve?

Reply:
don't remember.  Someone told me it fixed some problem, but if you don't think so I can remove those changes.